### PR TITLE
Update configs with new GitHub labels

### LINKED ISSUE #10473: Revamp GitHub labels
## 🚀 Feature

- [x] 1. Add/edit/remove labels
- [ ] 2. Check/update configs which uses outdated labels
  - https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/mergify.yml
  - https://github.com/PyTorchLightning/pytorch-lightning/tree/master/.github/ISSUE_TEMPLATE
  - https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/stale.yml

### Motivation
To address the linked issues.

### Pitch

https://docs.google.com/document/d/1JeCC0PadWVD84wYCBMRyParDpdz3oMfhJwQFrp1CD0c/edit?usp=sharing

### Additional context
This issue should be addressed before:
- #9519
- #9216

______________________________________________________________________

#### If you enjoy Lightning, check out our other projects! ⚡

<sub>

- [**Metrics**](https://github.com/PyTorchLightning/metrics): Machine learning metrics for distributed, scalable PyTorch applications.

- [**Lite**](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html): enables pure PyTorch users to scale their existing code on any kind of device while retaining full control over their own loops and optimization logic.

- [**Flash**](https://github.com/PyTorchLightning/lightning-flash): The fastest way to get a Lightning baseline! A collection of tasks for fast prototyping, baselining, fine-tuning, and solving problems with deep learning.

- [**Bolts**](https://github.com/PyTorchLightning/lightning-bolts): Pretrained SOTA Deep Learning models, callbacks, and more for research and production with PyTorch Lightning and PyTorch.

- [**Lightning Transformers**](https://github.com/PyTorchLightning/lightning-transformers): Flexible interface for high-performance research using SOTA Transformers leveraging Pytorch Lightning, Transformers, and Hydra.

</sub>

### PR DESCRIPTION
**Pull request by akihironitta**
## What does this PR do?
Fixes #10473.

- Update configs which uses outdated labels
  - https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/mergify.yml
  - ~https://github.com/PyTorchLightning/pytorch-lightning/tree/master/.github/ISSUE_TEMPLATE~
  - https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/stale.yml

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [n/a] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [n/a] Did you verify new and **existing tests pass** locally with your changes?
- [n/a] Did you list all the **breaking changes** introduced by this pull request?
- [n/a] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @carmocca

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -8,8 +8,9 @@ issues:
     daysUntilClose: 7
     # Issues with these labels will never be considered stale
     exemptLabels:
-      - Important
-      - Priority
+      - p0
+      - p1
+      - p2
```
Comment (akihironitta) [inline .github/stale.yml:13]: Should we remove `p2` label from here?
```suggestion
      - p0
      - p1
```

**Review by carmocca — APPROVED** [review]

**Comment by akihironitta** [comment]
Reverted the change in issue templates as #10537 changes it.

**Review by justusschock — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGTM !
